bash/common.sh

```diff
@@ -24,7 +24,8 @@ init()
 			# Don't upload packages at the end
 			--simulate-dput) dput_simulate="-s" ;;
 			--package-version-extension) shift; package_version_extension=$1 ;;
-			--package-name-suffix) shift; package_name_suffix=$1 ;;
+			# Suite location: eg. experimental, updates, proposed, dictionary
+			--suite-name) shift; suite-name=$1 ;;
```

Comment:
I don't think you can have a variable with a dash in its name, so it would be better named `suite_name`.